src/App.js

```diff
@@ -1,12 +1,18 @@
 import React, { Component } from 'react';
+import './app.scss';
+import { Button } from 'carbon-components-react';
+import { Content } from 'carbon-components-react/lib/components/UIShell';
+import TutorialHeader from './components/TutorialHeader';
 
 class App extends Component {
   render() {
     return (
-      <div>
-        Hello Carbon! Well, not quite yet. This is the starting point for the
-        Carbon tutorial.
-      </div>
+      <>
+        <TutorialHeader />
+        <Content>
+          <Button>Button</Button>
```

Comment:
The button component should be replaced with the Switch component: See https://www.carbondesignsystem.com/tutorial/react-step-1#add-routing